src/Controls/src/Core/HandlerImpl/Rectangle.Impl.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls.Shapes
+{
+	public partial class Rectangle : IShape
+	{
+		public PathF GetPath()
+		{
+			var path = new PathF();
+
+			path.AppendRoundedRectangle(0f, 0f, (float)Width, (float)Height, (float)RadiusY + (float)RadiusX);
```

Comment:
For now this is wrong anyway since the old behaviour for Rectangle is different. Needs some more math to curve for the x and y radius values. I was just shortcutting for now.